satellite/referral/offersweb/pages/404.html

```diff
@@ -0,0 +1,3 @@
+{{define "main"}}
```

Comment:
You may want to reuse the page from the Satellite UI

Reply:
It doesn't look like their is a 404 page for the satellite UI